src/prg_normalize_mod.F90

```diff
@@ -128,12 +129,14 @@ end subroutine prg_gershgorinReduction
   !! \param h_bml Input/Output Hamiltonian matrix
   subroutine prg_normalize_cheb(h_bml,mu,emin,emax,alpha,scaledmu)
 
+    implicit none
     type(bml_matrix_t),intent(inout) :: h_bml
 
     real(dp) :: beta, maxMinusMin
     real(dp), intent(in) :: mu, emin, emax
     real(dp), intent(inout) :: scaledmu, alpha
 
+    stop
```

Comment:
Is that a left over debug statement, or do you want to leave it here?

Reply:
Good catch Nick, thanks!